src/OpenApi/src/Services/OpenApiDocumentService.cs

```diff
@@ -90,7 +91,11 @@ public async Task<OpenApiDocument> GetOpenApiDocumentAsync(IServiceProvider scop
         document.Workspace.RegisterComponents(document);
         if (document.Components?.Schemas is not null)
         {
-            document.Components.Schemas = new SortedDictionary<string, IOpenApiSchema>(document.Components.Schemas);
+            // Sort schemas by key name for better readability and consistency
+            // This works around an API change in OpenAPI.NET
+            document.Components.Schemas = new Dictionary<string, IOpenApiSchema>(
+                document.Components.Schemas.OrderBy(kvp => kvp.Key),
```

Comment:
I don't think this does anything once you insert the items into the dictionary?

Reply:
This is done as the final step before returning the document to the consumer. The expectation is that new schemas aren't inserted into the dictionary after that. We could use FrozenDictionary/ImmutableDictionary to help codify these but the new APIs use `Dictionary` instead of `IDictionary` so we're constrained here. :/